introduction.md

```diff
@@ -10,3 +10,8 @@ Aha! I had to mention Code School path!
 But Are you realize mention about pay path?
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?